src/libaktualizr/storage/invstorage.h

```diff
@@ -57,6 +58,9 @@ class StorageTargetWHandle {
     }
     return is;
   }
+
+ protected:
+  size_t written_size_{0};
```

Comment:
I don't think it's a new issue but we should probably make a pass to make sure that we store full sizes of files with 64 bits on all architectures.